src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public abstract class Figure implements Drawable {
```

Comment:
According to the checklist, don't begin class or method implementation with an empty line. The empty line on line 2 should be removed.